### PR TITLE
Convert set ga code to our module format

### PR DESCRIPTION
**Pull request by andysellick**
⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

## What

Converts the set ga id on form code to use our standard JS modules format. This is used on https://www.gov.uk/check-income-tax-current-year/sign-in/prove-identity

## Why

We're trying to standardise our approach to JS and having it in our modules format means we also don't need to manually initialise this code in `application.js` anymore.

## Visual changes
None.

Trello card: https://trello.com/c/bqdisKbm/277-from-jquery-from-government-frontend-set-ga-client-id-on-formjs-and-applicationjs-moderate-js-knowledge-required

### CODE REVIEW

File: spec/javascripts/set-ga-client-id-on-form.spec.js [hunk 1]
```diff
@@ -11,7 +11,8 @@ describe('SetGaClientIdOnForm', function () {
     form = $(
       '<form class="js-service-sign-in-form" action="/endpoint"></form>'
     )
-    new GOVUK.SetGaClientIdOnForm(form[0]) // eslint-disable-line no-new
+    var setId = new GOVUK.Modules.SetGaClientIdOnForm(form[0]) // eslint-disable-line no-new
```
Comment (kevindew) [inline spec/javascripts/set-ga-client-id-on-form.spec.js:14]: You don't need to disable this linting rule anymore 🎉

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -10,10 +10,3 @@
 //= require govuk_publishing_components/components/step-by-step-nav
 
 //= require_tree ./modules
-//= require set-ga-client-id-on-form
-
-var form = document.querySelector('.js-service-sign-in-form')
```
Comment (kevindew) [inline app/assets/javascripts/application.js:15]: I'm wondering if this class can be removed from the HTML now? From a quick search I think it was only for this feature.

**Review by kevindew — COMMENTED** [review]
Looks great Andy ⭐  - just a couple of small comments.

File: app/assets/javascripts/application.js (hunk 2 above)
Comment (andysellick) [reply app/assets/javascripts/application.js:15]: Of course, thanks 🤦

**Review by kevindew — APPROVED** [review]
🕺

**Pull request merged**